Products/Zuul/catalog/indexable.py

```diff
@@ -322,6 +331,50 @@ def idx_deviceOrganizers(self):
                 organizers.append( "/".join(sp[:-2]) )
         return organizers
 
+    @indexed(StringFieldType(indexed=False, stored=True), attr_query_name="tagNumber")
```

Comment:
I thought we were going to make all these fields indexed?

Reply:
Hmm, we discussed that and from what I remember decided to have them only stored. If you say that we can have them indexed I can change that.